### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by CJentzsch**
inspired by the security audit

### CODE REVIEW

File: test/TestHelper.cpp [hunk 1]
```diff
@@ -262,7 +262,21 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
...
 		RLP transactionRLP(transactionRLPStream.out());
-		m_transaction = Transaction(transactionRLP.data(), CheckTransaction::Everything);
+		try
+		{
+			m_transaction = Transaction(transactionRLP.data(), CheckTransaction::Everything);
+		}
+		catch(InvalidSignature)
+		{
+			// create unsigned transaction
+			m_transaction = _o["to"].get_str().empty() ?
```
Comment (winsvega) [inline test/TestHelper.cpp]: have you tried ttTransactionTests after this changes?
please refill tests with --checkstate option

Comment (CJentzsch) [inline test/TestHelper.cpp]: yes, works.

File: test/TestHelper.cpp [hunk 2]
```diff
@@ -262,7 +262,21 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	{
 		RLPStream transactionRLPStream = createRLPStreamFromTransactionFields(_o);
 		RLP transactionRLP(transactionRLPStream.out());
-		m_transaction = Transaction(transactionRLP.data(), CheckTransaction::Everything);
+		try
+		{
+			m_transaction = Transaction(transactionRLP.data(), CheckTransaction::Everything);
+		}
+		catch(InvalidSignature)
```
Comment (gavofyork) [inline test/TestHelper.cpp]: space:
`catch (...`

File: test/TestHelper.cpp [hunk 3]
```diff
@@ -262,7 +262,21 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
...
+		{
+			// create unsigned transaction
+			m_transaction = _o["to"].get_str().empty() ?
+				Transaction(toInt(_o["value"]), toInt(_o["gasPrice"]), toInt(_o["gasLimit"]), importData(_o), toInt(_o["nonce"])) :
+				Transaction(toInt(_o["value"]), toInt(_o["gasPrice"]), toInt(_o["gasLimit"]), Address(_o["to"].get_str()), importData(_o), toInt(_o["nonce"]));
+		}
+		catch(Exception& _e)
```
Comment (gavofyork) [inline test/TestHelper.cpp]: space after `catch`

**Pull request merged**